sim/warrior/slam.go

```diff
@@ -44,7 +44,7 @@ func (warrior *Warrior) RegisterSlamSpell() {
 
 		// TODO: check if the OH SMF hit is on a separate attack table roll
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			baseDamage := 431 + 1.1*spell.Unit.MHWeaponDamage(sim, spell.MeleeAttackPower())
+			baseDamage := 625 + 1.25*spell.Unit.MHWeaponDamage(sim, spell.MeleeAttackPower())
```

Comment:
Same comment as above. Fix how you apply the weapon damage % and udpate to the correct base value (using base scaling * coeff instead of a hardcoded precomputed number